### DISCUSSION TITLE
在 Apple M1 平台搭建 Ubuntu 开发环境 | SF-Zhou's Blog

### DISCUSSION THREAD

**Discussion by giscus** [SF-Zhou/sf-zhou.github.io#210] [Announcements]
# 在 Apple M1 平台搭建 Ubuntu 开发环境 | SF-Zhou's Blog

https://sf-zhou.github.io/productivity/ubuntu_env_on_apple_m1.html

**Comment by SF-Zhou** [comment]
最近发现 [UTM](https://mac.getutm.app/gallery) 更好用，multipass 版本升级的时候容易抽风

**Reply by SF-Zhou** [reply]
需要 x86 开发环境时，推荐 [OrbStack](https://orbstack.dev)
